src/daal4py.cpp

```diff
@@ -662,3 +662,17 @@ bool c_assert_all_finite(const data_or_file & t, bool allowNaN, char dtype)
     return false;
 #endif
 }
+
+void c_train_test_split(data_or_file & orig, data_or_file & train, data_or_file & test,
+                        data_or_file & train_idx, data_or_file & test_idx)
+{
+#if INTEL_DAAL_VERSION >= 20200002
+    auto origTable = get_table(orig);
+    auto trainTable = get_table(train);
+    auto testTable = get_table(test);
+    auto trainIdxTable = get_table(train_idx);
+    auto testIdxTable = get_table(test_idx);
+    daal::data_management::internal::trainTestSplit<int>(*origTable, *trainTable, *testTable, *trainIdxTable, *testIdxTable);
```

Comment:
Why template `int`, not `float` and `double`?

Reply:
int is IndexType